grc/gui/Actions.py

```diff
@@ -646,10 +646,10 @@ def connect(action, handler=None):
                                  label='Reload _Blocks',
                                  tooltip='Reload Blocks',
                                  icon_name='view-refresh')
-FIND_BLOCKS = actions.register(
-    "win.find_blocks",
-    label='_Find Blocks',
-    tooltip='Search for a block by name (and key)',
+TOGGLE_SEARCH = actions.register(
+    "win.toggle_search",
+    label='_Toggle Search',
```

Comment:
In the menu, this is not particularly clear.  Maybe "Show Block Search" and have it be a checkbox - would like some feedback from others on this